### DISCUSSION TITLE
Compiling oqs with NIST standardized schemes only

### DISCUSSION THREAD

**Discussion by yamus-didine** [open-quantum-safe/liboqs#1529] [Q&A] [answered]
Hello.
I am trying to compile oqs to include NIST standardized schemes only.
I issued this `cmake` command:

`cmake -GNinja -DOQS_USE_OPENSSL=ON -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DOQS_ALGS_ENABLED=STD -DOQS_BUILD_ONLY_LIB=ON ..`

I got the following log:

![oqs_compile](https://github.com/open-quantum-safe/liboqs/assets/134303261/5638909d-d118-4bbe-adf1-c7bdc386fc0a)

I issued the same `cmake` command again. I got the following log:

![oqs_compile2](https://github.com/open-quantum-safe/liboqs/assets/134303261/b6adca82-1639-4da0-a63c-b65da6b7dede)

Is there something wrong?

**Comment by baentsch** [comment, marked as answer]
> Is there something wrong?

Yes: You are using an outdated `liboqs` release. Please try with v0.8.0 (or main branch): The log output indicates that "OQS_ALGS_ENABLED" is an unused/unsupported variable in that code base.

**Reply by baentsch** [reply]
Does the thumps up sign mean this should be marked as an answer then (to close the discussion)?

**Reply by yamus-didine** [reply]
Hello
Yes, actually it worked as you indicated.
I forgot to mark the thread as answered.
I appreciate the help.
